instance/models/mixins/openedx_config.py

```diff
@@ -43,6 +43,12 @@ def _get_configuration_variables(self):
             # Use database-backed sessions to stay logged in across appserver deploys.
             # (We have EDXAPP_CLEARSESSIONS_CRON_ENABLED to clean up old data.)
             "SESSION_ENGINE": "django.contrib.sessions.backends.cached_db",
+            # The following two settings help determine the public IP of the request.
+            # See: https://discuss.openedx.org/t/security-patch-for-rate-limiting-in-edx-platform/7078
+            "CLOSEST_CLIENT_IP_FROM_HEADERS": [
+                {"name": "X-Forwarded-For", "index": -1},
+            ],
+            "NUM_PROXIES": 1,
```

Comment:
@mtyaka should this not be part of the `REST_FRAMEWORK` settings?

Quoting from the [docs](https://www.django-rest-framework.org/api-guide/settings/)
> Configuration for REST framework is all namespaced inside a single Django setting, named REST_FRAMEWORK.

Reply:
Thanks for catching that @keithgg, you are absolutely right. I'll update the code and reprovision the appserver.